app/(dashboard)/item/[id]/page.tsx

```diff
@@ -53,7 +54,14 @@ const DisplayItemDetails = async ({ params }: { params: { id: string } }) => {
               postage_covered={item.postage_covered}
             />
             <div className='min-h-40 w-full bg-secondaryGray p-10 md:w-1/2 md:rounded-lg'>
-              <h2 className='text-xl italic'>{item.item_name}</h2>
+              <div className='flex flex-row content-start justify-between'>
+                <h2 className='text-xl italic'>{item.item_name}</h2>
+                {item.reserved && (
+                  <p className='-m-2 rounded-full bg-primaryOrange p-3'>
+                    Reserved
+                  </p>
```

Comment:
Really simple solution 👌 

I know we don't have a design for this flag at the moment so there was no guidance in the issue but can you try to remove the background and just make the font this colour? Maybe make it bold if it needs, I'm not sure.

We've just gained some UI people so there will be a proper redesign of things soon but in the meantime I think if we simplify this one it will just blend in very nicely with the existing style of the cards and hold better until we have to rework the overall image of kindly

Reply:
I sure can, no problem. 😃